packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -205,3 +205,8 @@ String getAssetBuildDirectory() {
 String getIosBuildDirectory() {
   return fs.path.join(getBuildDirectory(), 'ios');
 }
+
+/// Returns directory used by incremental compiler to store cached artifacts.
+String getIncrementalCompilerByteStoreDirectory() {
+  return fs.path.join(getBuildDirectory(), 'ikg_byte_store');
+}
```

Comment:
seems to be missing newline at end of file.

Reply:
Done